packages/adapter/addon/rest.js

```diff
@@ -516,7 +516,6 @@ const RESTAdapter = Adapter.extend(BuildURLMixin, {
     @method findAll
     @param {DS.Store} store
     @param {DS.Model} type
-    @param {String} sinceToken
```

Comment:
The argument slot continues to exist as the SnapshotArray is the fourth argument passed in.

```suggestion
    @param {undefined} neverSet a value is never provided to this argument
```